olp-cpp-sdk-authentication/include/olp/authentication/AuthenticationError.h

```diff
@@ -26,30 +26,43 @@
 
 namespace olp {
 namespace authentication {
+
+/**
+ * @brief Contains information on the authentication error.
+ *
+ * You can get the following information on the authentication error: the error
+ * code and error message.
```

Comment:
```suggestion
 * code (\ref GetErrorCode ) and error message ( \ref GetMessage ).
```
Maybe better add links to the accessors of those information?

Reply:
Added (here and in the other files).